### ISSUE TITLE
Training schedule

### ISSUE THREAD

**Issue by konyul** [Sranc3/M-BEV#3]
Thank you for sharing such a great work!
I have some questions for the training details.
1. In the paper, 48 epochs are consumed to train MVR, however, in the github, 24 epoch are consumed for finetuning. which one is right?
2. how many epochs do you consume for pretraining?

**Comment by Sranc3** [comment]
Thanks for your interest. For one missing camera, finetuning for 24 epochs could achieve most of the performance, and for 48 epochs the performance will be saturated. So you can decide how many epochs you want to train with time cost in mind. Note, for more missing views, more training epochs are recommended. In the experiment, the pretraining time is too long while the improvement is limited compared to finetuning, and we suggest to have a short pretraining for rough initialization or go through the finetuning with mvr loss supervision directly. You can have your own try and find the suitable schedule according to your demand.

**Comment by konyul** [comment]
Thank you for the answer, I am satisfied with your answer.
